examples/VLCB_4in4out/VLCB_4in4out.ino

```diff
@@ -132,7 +132,7 @@ void setupVLCB()
   modconfig.EE_EVENTS_START = 50;
   modconfig.EE_MAX_EVENTS = 64;
   modconfig.EE_PRODUCED_EVENTS = NUM_SWITCHES;
-  modconfig.EE_NUM_EVS = NUM_LEDS;
+  modconfig.EE_NUM_EVS = 1 + NUM_LEDS;
```

Comment:
Hmm, this is dangerous. Now the user sketch has to take the default produced event variable into account. What if the user forgets this?

Reply:
I also don't like having random numbers scattered about.  I was thinking of re-introducing the GLOBAL_EVS that we had at one stage in the CBUS version for handling SoD.  Alternatively, there could be a constant byte SYSTEM_EVS = 1.  Let's discuss later.

Reply:
Happy to leave this for now. We can improve this configuration later.

Reply:
OK.  I'll sort it next time round and certainly before we go to a wider audience